Sources/Utils.php

```diff
@@ -440,18 +441,18 @@ public static function htmlspecialchars(string $string, int $flags = ENT_COMPAT,
 	 *
 	 * Only affects values.
 	 *
-	 * @param array|string $var The string or array of strings to add entities to
+	 * @param mixed $var The string or array of strings to add entities to
 	 * @param int $flags Bitmask of flags to pass to standard htmlspecialchars().
 	 *    Default is ENT_COMPAT.
 	 * @param string $encoding Character encoding. Default is UTF-8.
 	 * @return array|string The string or array of strings with entities added
 	 */
-	public static function htmlspecialcharsRecursive(array|string $var, int $flags = ENT_COMPAT, string $encoding = 'UTF-8'): array|string
+	public static function htmlspecialcharsRecursive(mixed $var, int $flags = ENT_COMPAT, string $encoding = 'UTF-8'): array|string
```

Comment:
This should not be expanded to accept mixed. Our wrapper should handle the validation of string type, or cast $var/$v to string as needed before passing it.

Reply:
It's receiving ints/float/other basic types as well.  Seems easier to let it just return those (we don't need to handle it).

Reply:
In this case I can see that. Lets just not use it as a crutch to not clean up the related calling code as needed when we can.

Reply:
Affirmative.  It was easier to use mixed than define a long list of types.  I believe only strings need cleaning through `htmlspecialchars`, but since we accept an array, we may have other types being processed in it due to recursion.  If anyone knows of other types that also need cleansing, let me know so we can do those.